src/views/List.jsx

```diff
@@ -15,6 +18,19 @@ export function List({ data, listToken }) {
 		}
 	}, [data, itemSearch]);
 
+	useEffect(() => {
+		async function sortShoppingList() {
+			const sortedData = await comparePurchaseUrgency(searchedData);
+			try {
+				setNewData(sortedData);
+			} catch (e) {
+				setSortErrorMessage('There was an error sorting your list');
+			}
+		}
+
+		sortShoppingList();
+	}, [searchedData]);
```

Comment:
There may be some cleanup we can do here! 

What's the reason behind using a `useEffect`? & do we need this to be `async`? & what error do you foresee happening from `comparePurchaseUrgency`?